Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -314,12 +415,36 @@ private void Awake()
             moveLogic = new ManipulationMoveLogic();
             rotateLogic = new TwoHandRotateLogic();
             scaleLogic = new TwoHandScaleLogic();
+
+            InitializeElastics();
         }
         private void Start()
         {
             rigidBody = HostTransform.GetComponent<Rigidbody>();
             constraints = new ConstraintManager(gameObject);
         }
+        private void Update()
+        {
+            // If the user is not actively interacting with the object,
+            // we let the elastic systems continue simulating, to allow
+            // the object to naturally come to rest.
+            if (!isManipulationStarted)
+            {
+                if (elasticTypes.HasFlag(TransformFlags.Move) && translationElastic != null && translationElastic.GetCurrentVelocity().magnitude > elasticVelocityThreshold)
```

Comment:
Will only continue simulating the spring if the user is not actively manipulating the object, if the elastic is non-null, if that particular type of elastic is enabled, and if the current elastic velocity is above the threshhold.

Reply:
Do you have a sense of the impact/cost of evaluatijng the magnitude or each of these things on an ongoing/passive basis? i.e. if the scene starts, I'm assuming this doesn't do anything because translationElastic might be nul at this point, but after manipulation has happened, I assume this will run on every update loop.

Reply:
Good point. I put the `HasFlag` check first, because if the user has not selected elastic to be enabled, this check should short-circuit and the magnitude should never be evaluated.

The checks are ordered from "least expensive" to "most expensive". The magnitude will never be evaluated if the elastic is not enabled.